core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -1384,6 +1384,17 @@ public static double atan2(double b0, double b1) {
     return Math.atan2(b0, b1);
   }
 
+  // CBRT
+  /** SQL <code>CBRT</code> operator applied to BigDecimal values. */
+  public static double cbrt(BigDecimal b0) {
+    return Math.cbrt(b0.doubleValue());
+  }
+
+  /** SQL <code>CBRT</code> operator applied to double values. */
+  public static double cbrt(double b0) {
+    return Math.cbrt(b0);
+  }
```

Comment:
Maybe we don't need to name the parameter as `b0` since there is only one parameter. I think `b` is fine.

Reply:
> Maybe we don't need to name the parameter as `b0` since there is only one parameter. I think `b` is fine.

well, I have changed it.